sapi/fuzzing/compat.h

```diff
@@ -0,0 +1,6 @@
+#ifndef TSRMLS_C
+#define TSRMLS_C
+#define TSRMLS_CC
+#define TSRMLS_DC
+#define TSRMLS_FETCH()
+#endif
```

Comment:
As mentioned before, this header should be dropped and uses of `TSRMLS_*` in code removed.